Automattic/wp-calypso
# (Do not merge) Rename FSE plugin

#### Changes proposed in this Pull Request
* Rename FSE plugin to "WordPress.com Editing Experience"
* Change directory from `apps/full-site-editing/full-site-editing-plugin` to `apps/wpcom-editing-experience/plugin`
* Rename `full-site-editing-plugin.php` to just `index.php`
* Update all build/dev tooling to point to new directory

#### Testing instruction

Note: you must now use `npx lerna run dev --scope='@automattic/wpcom-editing-experience' --stream` to build the plugin

1. All CircleCI stuff should work as expected
2. Smoke test everything
3. Search for mentions of FSE in the codebase and see if I missed anything

Wpcom: apply D36573-code and use `./bin/calypso/build-wpcom-editing-experience $job_id 0.18` with the job ID from _build-wpcom-editing-experience_ in the circle CI build
